src/main/java/com/solvd/movie/web/dto/mapper/MovieMapper.java

```diff
@@ -6,11 +6,14 @@
 import org.mapstruct.Mapper;
 
 @Mapper(componentModel = "spring")
-public interface MovieMapper
-        extends com.solvd.movie.web.dto.mapper.Mapper<PgMovie, MovieDto> {
+public interface MovieMapper {
 
-    EsMovie toEntity(PgMovie movie);
+    PgMovie toEntity(MovieDto dto);
```

Comment:
@hizmailovich you can declare one interface based on `Movie transform(Movie m)`
and implement this interface by using Pg -> Elastic and Elastic -> Pg mapping